### PR TITLE
13860 process statuses filters

### PR DESCRIPTION
**Pull request by kowalczykpawel7**

### CODE REVIEW

File: src/views/process-panel/subprocesses-card.tsx [hunk 1]
```diff
@@ -4,39 +4,60 @@
...
+type CssRules = 'root' | 'subtitle' | 'title' | 'gridFilter';
 
 const styles: StyleRulesCallback<CssRules> = (theme: ArvadosTheme) => ({
     root: {
-        fontSize: '0.875rem'
+        fontSize: '0.875rem',
+        height: '100%'
+    },
+    subtitle: {
+        paddingBottom: '32px!important'
```
Comment (janicka1) [inline src/views/process-panel/subprocesses-card.tsx]: I suggest you don't use `important`

File: src/views/process-panel/subprocesses-card.tsx [hunk 2]
```diff
@@ -4,39 +4,60 @@
...
+    },
+    title: {
+        color: theme.customs.colors.grey700
+    },
+    gridFilter: {
+        height: '20px',
+        marginBottom: theme.spacing.unit,
+        paddingTop: '0px!important',
```
Comment (janicka1) [inline src/views/process-panel/subprocesses-card.tsx]: I suggest you don't use important

File: src/views/process-panel/process-information-card.tsx [hunk 3]
```diff
@@ -94,20 +94,25 @@ export const ProcessInformationCard = withStyles(styles, { withTheme: true })(
...
-                           {process.containerRequest.name}
+                    <Tooltip title={process.containerRequest.name} placement="bottom-start">
+                        <Typography noWrap variant="title" color='inherit'>
+                            {process.containerRequest.name}
                         </Typography>
                     </Tooltip>
                 }
-                subheader={process.containerRequest.description} />
+                subheader={
+                    <Tooltip title={process.containerRequest.description || '(no-description)'} placement="bottom-start">
```
Comment (janicka1) [inline src/views/process-panel/process-information-card.tsx]: You could create a method and return description or default value. 
You duplicate a code here.

File: src/common/formatters.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@
 //
 // SPDX-License-Identifier: AGPL-3.0
 
-export const formatDate = (isoDate?: string) => {
+export const formatDate = (isoDate?: string | null) => {
```
Comment (unodgs) [inline src/common/formatters.ts]: isoDate already can be undefined. No need for null here

File: src/store/process-panel/process-panel-actions.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+export const toggleProcessPanelFilter = procesPanelActions.TOGGLE_PROCESS_PANEL_FILTER;
+
+export const loadProcessPanel = (uuid: string) =>
+    (dispatch: Dispatch) => {
+        dispatch<any>(loadProcess(uuid));
+        dispatch(initProcessPanelFilters);
+    };
+
+export const initProcessPanelFilters = procesPanelActions.INIT_PROCESS_PANEL_FILTERS([
```
Comment (unodgs) [inline src/store/process-panel/process-panel-actions.ts]: Why this has to be initialized in reducer? Why cannot it be set as initialState? If you want to be set by reducer that change name to `SET_PROCESS_PANEL_FILTERS`

File: src/common/formatters.ts (hunk 4 above)
Comment (burzomir) [reply src/common/formatters.ts]: @unodgs  
Some values - especially those coming from api - can be `null` so it had to be done this way.
```isoDate?: string === isoDate: string | undefined```

File: src/store/process-panel/process-panel-actions.ts (hunk 5 above)
Comment (burzomir) [reply src/store/process-panel/process-panel-actions.ts]: I agree on changing name to SET_PROCESS_PANEL_FILTERS. But IMO initialization data should stay in action.

**Pull request merged**